include/TrackContainer.h

```diff
@@ -41,6 +41,12 @@ class EXPORT TrackContainer : public Model, public JournallingObject
 {
 	Q_OBJECT
 public:
+	enum TrackContainerTypes
+	{
+		BBTrackContainer,
```

Comment:
@Lukas-W, do we have a convention established for enums?  Most of our code uses camelcase, which I find to be tremendously confusing (they tend to behave more like constants).  `CamelCase` generally suggests an Object/Class and `camelCase` or `under_score` suggests a variable, right?

It's nothing wrong with Dan's code, but perhaps this is the time to establish a convention to use?